examples/lm1b/train.py

```diff
@@ -450,34 +453,34 @@ def encode_strings(strs, max_len):
   initial_variables = jax.jit(m.init)(init_rng,
                                       jnp.ones(input_shape, jnp.float32))
 
-  # apply an optimizer to this tree
-  optimizer_def = optim.Adam(
-      config.learning_rate,
-      beta1=0.9,
-      beta2=0.98,
-      eps=1e-9,
-      weight_decay=config.weight_decay)
-  optimizer = optimizer_def.create(initial_variables["params"])
+  learning_rate_fn = create_learning_rate_scheduler(
```

Comment:
I think we should update the `create_learning_rate_scheduler` like here:

https://github.com/google/flax/blob/4b1baddfb93fd7c7f957ce4cc10e5a0844e78030/examples/wmt/train.py#L47-L74

Reply:
Done.